### PR TITLE
[#59] Create 404 page + [#62] Create 403 page

### PR DESCRIPTION
**Pull request by semyonov-m-y**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/ContactSupport.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/ContactSupport.html]: MAJOR Add a 'favicon' declaration in this 'header' tag. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

File: src/main/webapp/WEB-INF/views/403Error.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/403Error.html]: MAJOR Add a 'favicon' declaration in this 'header' tag. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

File: src/main/webapp/WEB-INF/views/403Error.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="row">
+    <div class="error-template">
+		<h1>Oops!</h1>
+	    <h2>403 Forbidden</h2>
+	    <div class="error-details">
+		Access to this resource is denied!<br>
+	    </div>
+	    <div class="error-actions">
+		<a href="DesignPage.html" class="btn btn-primary">
+		    <i class="icon-home icon-white"></i> Take Me Home </a>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/403Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/WEB-INF/views/403Error.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+		<a href="ContactSupport.html" class="btn btn-default">
+		    <i class="icon-envelope"></i> Contact Support </a>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/403Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/WEB-INF/views/404Error.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/404Error.html]: MAJOR Add a 'favicon' declaration in this 'header' tag. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

File: src/main/webapp/WEB-INF/views/404Error.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="row">
+    <div class="error-template">
+	    <h1>Oops!</h1>
+	    <h2>404 Not Found</h2>
+	    <div class="error-details">
+		Sorry, an error has occured, Requested page not found!<br>
+	    </div>
+	    <div class="error-actions">
+		<a href="DesignPage.html" class="btn btn-primary">
+		    <i class="icon-home icon-white"></i> Take Me Home </a>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/404Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/WEB-INF/views/404Error.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+		<a href="ContactSupport.html" class="btn btn-default">
+		    <i class="icon-envelope"></i> Contact Support </a>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/404Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 7 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by Sheptytskyid — COMMENTED** [review]
Пропоную відразу у цьому таску зробити мапінг на ці html файли, або перетягнути файли у webapp/error/, щоб вони автоматично відображалися при виникненні помилки (без мапінга).

File: src/main/webapp/WEB-INF/views/403Error.html (hunk 2 above)
Comment (GetFire) [inline src/main/webapp/WEB-INF/views/403Error.html]: Добавь фавикон. Он лежит в папке resources/img

File: src/main/webapp/WEB-INF/views/404Error.html (hunk 5 above)
Comment (GetFire) [inline src/main/webapp/WEB-INF/views/404Error.html]: Добавь фавикон. Он лежит в папке resources/img

**Comment by GetFire** [comment]
Поддреживаю Дениса! И обнови ветку

File: src/main/webapp/WEB-INF/views/ContactSupport.html [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse"
+                    data-target="#responsive-menu">
+                <span class="sr-only">Menu</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/ContactSupport.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/WEB-INF/views/ContactSupport.html [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+    </div>
+</div>
+
+<div class="modal fade" id="modal-login">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Registration
+                    <button class="close" type="button" data-dismiss="modal">
+                        <i class="fa fa-close"></i>
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/ContactSupport.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/WEB-INF/views/ContactSupport.html [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+                        <input type="text" class="form-control" placeholder="login" value="">
+                    </div>
+                    <div class="form-group">
+                        <input type="password" class="form-control" placeholder="password" value="">
+                    </div>
+                    <button type="submit" class="btn btn-success">Login</button>
+
+                    <form action="/registration">
+                        <button class="btn btn-primary" type="submit">
+                            <i class="fa fa-user-circle-o"></i> Create an account
```
Comment (Sheptytskyid) [inline src/main/webapp/WEB-INF/views/ContactSupport.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/404Error.html [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse"
+					data-target="#responsive-menu">
+				<span class="sr-only">Menu</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
```
Comment (Sheptytskyid) [inline src/main/webapp/error/404Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/404Error.html [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+</div>
+</div>
+
+<div class="modal fade" id="modal-login">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<h4 class="modal-title">Registration
+					<button class="close" type="button" data-dismiss="modal">
+						<i class="fa fa-close"></i>
```
Comment (Sheptytskyid) [inline src/main/webapp/error/404Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/404Error.html [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+						<input type="text" class="form-control" placeholder="login" value="">
+					</div>
+					<div class="form-group">
+						<input type="password" class="form-control" placeholder="password" value="">
+					</div>
+					<button type="submit" class="btn btn-success">Login</button>
+
+					<form action="/registration">
+						<button class="btn btn-primary" type="submit">
+							<i class="fa fa-user-circle-o"></i> Create an account
```
Comment (Sheptytskyid) [inline src/main/webapp/error/404Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/403Error.html [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse"
+					data-target="#responsive-menu">
+				<span class="sr-only">Menu</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
```
Comment (Sheptytskyid) [inline src/main/webapp/error/403Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/403Error.html [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+</div>
+</div>
+
+<div class="modal fade" id="modal-login">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<h4 class="modal-title">Registration
+					<button class="close" type="button" data-dismiss="modal">
+						<i class="fa fa-close"></i>
```
Comment (Sheptytskyid) [inline src/main/webapp/error/403Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/error/403Error.html [hunk 16]
```diff
@@ -0,0 +1,184 @@
...
+						<input type="text" class="form-control" placeholder="login" value="">
+					</div>
+					<div class="form-group">
+						<input type="password" class="form-control" placeholder="password" value="">
+					</div>
+					<button type="submit" class="btn btn-success">Login</button>
+
+					<form action="/registration">
+						<button class="btn btn-primary" type="submit">
+							<i class="fa fa-user-circle-o"></i> Create an account
```
Comment (Sheptytskyid) [inline src/main/webapp/error/403Error.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 11 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by GetFire** [comment]
http://stackoverflow.com/questions/37398385/spring-boot-and-custom-404-error-page
Почитай, попробуй сделать как там написано. Только нужно будет подумать как подключить ресурсы к тем html

File: src/main/webapp/error/403.html [hunk 17]
```diff
@@ -0,0 +1,184 @@
...
+
+		<div class="navbar-header">
+			<button type="button" class="navbar-toggle" data-toggle="collapse"
+					data-target="#responsive-menu">
+				<span class="sr-only">Menu</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+			<a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
```
Comment (Sheptytskyid) [inline src/main/webapp/error/403.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/webapp/error/403.html (hunk 17 above)
Comment (zozich) [inline src/main/webapp/error/403.html]: Replace with <span>

File: src/main/webapp/resources/css/error.css [hunk 18]
```diff
@@ -0,0 +1,15 @@
+.error-template {
+padding: 40px 15px;
```
Comment (zozich) [inline src/main/webapp/resources/css/error.css]: Where is the indent?

File: src/main/webapp/resources/js/contact-support.js [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+         }        
+	 $.ajax({
+                url: "./bin/contact_me.php",
+            	type: "POST",
+            	data: {name: name, email: email, message: message},
+            	cache: false,
+            	success: function() {  
+            	// Success message
+            	   $('#success').html("<div class='alert alert-success'>");
+            	   $('#success > .alert-success').html("<button type='button' class='close' data-dismiss='alert' aria-hidden='true'>&times;")
```
Comment (zozich) [inline src/main/webapp/resources/js/contact-support.js]: Why not just search for ".alert-success"?

**Review by zozich — APPROVED** [review]
See comments

**Pull request merged**